wallet/assets/countries.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
Place this file into a wallet/assets/wallofcoins/ folder so that it is clear that this is all for the Wall of Coins integration.